docs/cristin-proxy-swagger.yaml

```diff
@@ -759,14 +750,81 @@ components:
             - CONCLUDED
           description: Project status, based on start_date and end_date.
         academicSummary:
+          $ref: '#/components/schemas/Languages'
+        popularScientificSummary:
+          $ref: '#/components/schemas/Languages'
+    ProjectPost:
+      type: object
+      properties:
+        '@context':
+          $ref: '#/components/schemas/Context'
+        type:
+          type: string
+          pattern: 'Project'
+          description: The type of the returned object, always Project
+        title:
+          type: string
+          pattern: '.+'
+          description: Title of the project
+        language:
+          type: string
+          pattern: '^http:\/\/lexvo.org\/id\/iso639-3\/[a-z]{3}$'
+          description: The lexvo URI for the language of the Project
+        alternativeTitles:
+          type: array
+          description: Titles in alternative languages
+          items:
+            $ref: '#/components/schemas/Languages'
+        startDate:
           type: string
-          description: Summary intended for an academic audience
+          pattern: '.+'
+          description: Project start date
+        endDate:
+          type: string
+          pattern: '.+'
+          description: Project end date
+        coordinatingInstitution:
+          $ref: '#/components/schemas/Organization'
+        funding:
+          type: array
+          description: Fundings of this project
+          items:
+            $ref: '#/components/schemas/Funding'
+        contributors:
+          type: array
+          description: List of contributors on the project
+          items:
+            type: object
+            description: One contributor
+            properties:
+              type:
+                type: string
+                enum:
+                  - ProjectManager
+                  - ProjectParticipant
+                description: Type of contributor. Can be ProjectManager or ProjectParticipant
+              identity:
+                type: object
+                properties:
+                  id:
+                    $ref: '#/components/schemas/Id'
+                  type:
+                    type: string
+                    pattern: 'Person'
+                    description: Type of identity. Always Person
+              affiliation:
+                $ref: '#/components/schemas/Organization'
+        status:
```

Comment:
This is not needed. Cristin sets these automatically. But since we need it in proxy we have to keep it for now and change it later when we remove it as a required field